altair/vegalite/v5/api.py

```diff
@@ -3909,11 +3909,10 @@ def interactive(
 
 
 def _add_tooltip(chart: _TChart, /) -> _TChart:
-    mark = chart.mark
-    if isinstance(mark, str):
-        mark = {"type": mark, "tooltip": True}
+    if isinstance(chart.mark, str):
+        chart.mark = {"type": chart.mark, "tooltip": True}
```

Comment:
👍 

Glad to see you was spotted after testing @joelostblom .
I did a lot of renaming & aliasing, but this one was both invalid & didn't save any lines.